<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardCenter</class>
 <widget class="QMainWindow" name="BoardCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CAF6000 BOARD CENTER</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/board_center_BKG.jpg);</string>
   </property>
   <widget class="Wiper" name="wiper" native="true">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>484</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="anulacionEmergencia" native="true">
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>484</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="bypassTraccion" native="true">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>208</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="abrirIzquierda" native="true">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>357</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="cerrarIzquierda" native="true">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>357</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="selectorIzquierda" native="true">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>357</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="selectorDerecha" native="true">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>357</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="cerrarDerecha" native="true">
    <property name="geometry">
     <rect>
      <x>736</x>
      <y>357</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="abrirDerecha" native="true">
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>357</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="silbato" native="true">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>358</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SpeedGaugeLeds" name="velocimetro" native="true">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>16</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="bypassFreno" native="true">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>208</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="megafonia" native="true">
    <property name="geometry">
     <rect>
      <x>737</x>
      <y>485</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="iluminacionReducPulsad" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>632</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="antivaho" native="true">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>482</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="faros" native="true">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>483</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="iluminacionPulsadores" native="true">
    <property name="geometry">
     <rect>
      <x>357</x>
      <y>483</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="QQuickWidget" name="modoConduccion">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
    <property name="source">
     <url>
      <string>qrc:/mode_selector.qml</string>
     </url>
    </property>
   </widget>
   <widget class="SingleButton" name="luzCabina" native="true">
    <property name="geometry">
     <rect>
      <x>785</x>
      <y>631</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="publicoCabina" native="true">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>483</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="selectorPuertas">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>629</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resources/transparent);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="../subtewidgets/widgetsrsc.qrc">:/resources/selectordepuertas.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bypassFrenoTraccion">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>286</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>OO</string>
    </property>
   </widget>
   <zorder>velocimetro</zorder>
   <zorder>wiper</zorder>
   <zorder>anulacionEmergencia</zorder>
   <zorder>bypassTraccion</zorder>
   <zorder>abrirIzquierda</zorder>
   <zorder>cerrarIzquierda</zorder>
   <zorder>selectorIzquierda</zorder>
   <zorder>selectorDerecha</zorder>
   <zorder>cerrarDerecha</zorder>
   <zorder>abrirDerecha</zorder>
   <zorder>silbato</zorder>
   <zorder>bypassFreno</zorder>
   <zorder>megafonia</zorder>
   <zorder>iluminacionReducPulsad</zorder>
   <zorder>antivaho</zorder>
   <zorder>faros</zorder>
   <zorder>iluminacionPulsadores</zorder>
   <zorder>modoConduccion</zorder>
   <zorder>luzCabina</zorder>
   <zorder>publicoCabina</zorder>
   <zorder>selectorPuertas</zorder>
   <zorder>bypassFrenoTraccion</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>SingleButton</class>
   <extends>QWidget</extends>
   <header>singlebutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Wiper</class>
   <extends>QWidget</extends>
   <header location="global">wiper.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpeedGaugeLeds</class>
   <extends>QWidget</extends>
   <header>speedgaugeleds.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../subtewidgets/widgetsrsc.qrc"/>
  <include location="../subtewidgets/widgetsrsc.qrc"/>
 </resources>
 <connections/>
</ui>
